File: Spoj/bafo.c
#include <stdio.h>
int main()
{
    int x;
    int aldo;
    int beto;
    int a,b;
    int i;
    int t = 1;
    while(scanf("%d", &x) != 0)
    {
        if(x == 0)
            break;
        aldo = 0;
        beto = 0;
        for(i = 0; i < x; ++i)
        {
            scanf("%d %d", &a, &b);
            aldo += a;
            beto += b;
        }
        if(aldo > beto)
            printf("Teste %d\nAldo\n\n", t);
        else
            printf("Teste %d\nBeto\n\n", t);
        t++;
    }
    return 0;
}

File: Spoj/Implementacao/bafo.py
p = 1
caso = 1
while(p != 0):
    p = int(raw_input())
    if p == 0:
        break
    aldo = 0
    beto = 0
    i = 0
    while i < p:
        a,b = map(int, raw_input().split())
        aldo = aldo + a
        beto = beto + b
        i = i + 1
    if(aldo > beto):
        print("Teste %d\nAldo\n") %caso
    else:
        print("Teste %d\nBeto\n") %caso
    caso = caso + 1

